README.rst

```diff
@@ -198,7 +198,7 @@ Where can I report bugs and make feature requests?
 --------------------------------------------------
 
 This project is considered a community work. The development crew
-consist of you too. I am very thankful for pull request. So if you
+consists of you too. I am very thankful for pull request. So if you
```

Comment:
Make it `we are very thankful`, you know as its message from Org.

Reply:
As its message from Org, I have changed the entire paragraph into first person plural.